source/systems/MSU-HPC_user_guide.rst

```diff
@@ -1760,467 +1807,28 @@ Policies and Best Practices
    Once your HFS data is deleted it will NOT be recoverable. Project data
    (``/work`` and ``/work2``) is not deleted when a users account is deleted.
 
-.. rubric:: Best Practices
+**Best Practices**
```

Comment:
Heading.

But since the only item here is getting data off MSU, back to Niagara/HPSS the title should reflect that.